### ISSUE TITLE
Test - why don't you evaluate the loss function ?

### ISSUE THREAD

**Issue by LulaSan** [mrharicot/monodepth#260]
Hi! I was trying the test mode in your code and I figured out that you don't obtain as output the loss function, in order to see if it is the same as the one in training time. 
I have seen in the paper that you obtain disparity, convert into depth, and compare with GT. 
Why don't you calculate also the loss function? 

Thank you, 
Luisa
